### PR TITLE
Cross-namespace policy query; fix broken link in README

### PR DESCRIPTION
**Pull request by YFSHEN97**

### CODE REVIEW

File: pkg/reconciler/delivery/delivery.go [hunk 1]
```diff
@@ -143,7 +144,12 @@ func (c *Reconciler) fetchRevisions(cfg *v1.Configuration) (map[string]*v1.Revis
 func (c *Reconciler) fetchPolicy(cfg *v1.Configuration) (*Policy, error) {
 	// there's no need for defensive map query check, because it would have been taken care of in ReconcileKind
 	policyName := cfg.Annotations[delivery.PolicyNameKey]
-	p, err := c.policyLister.Policies(cfg.Namespace).Get(policyName)
+	policyNamespace := cfg.Namespace
+	if s := strings.SplitN(policyName, "/", 2); len(s) > 1 {
```
Comment (trshafer) [inline pkg/reconciler/delivery/delivery.go:148]: test

Comment (YFSHEN97) [reply pkg/reconciler/delivery/delivery.go:148]: It's due to a gofmt error, I've fixed it

Comment (trshafer) [reply pkg/reconciler/delivery/delivery.go:148]: This is new functionality, there should be a unit test that validates this functionality.

Comment (YFSHEN97) [reply pkg/reconciler/delivery/delivery.go]: Added, PTAL. `fetchPolicy` itself is hard to unit test, so I extracted the bits where policy namespace and name are extracted and tested that logic alone.

**Pull request merged**